test/didweb.go

```diff
@@ -0,0 +1,144 @@
+package test
+
+import (
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/mr-tron/base58"
+	"github.com/tbd54566975/ssi-service/internal/keyaccess"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func RunDidWebTest() error {
```

Comment:
I think all tests should have the `_test.go` suffix and we should use the `testing.T` paradigm.

For integration tests, let's take advantage of the [paradigm outlined here](https://stackoverflow.com/a/41407042) so we can selectively run unit/integration tests in our pipelines.

Reply:
Heavy +1 to this.

Reply:
did this in the refactor